https/README.md

```diff
@@ -0,0 +1,47 @@
+# Https Package for prometheus
+
+The `https` directory contains files and a template config for the implementation of tls.
+When running a server with tls use the flag `--web.tls-config`
+Where the path is from where the exporter was run.
+
+e.g. `./node_exporter --web.tls-config="https/tls-config.yml"`
+If the config is kept within the https directory 
+
+### TLS Config Layout
```

Comment:
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file has the syntax to use here